[Bug]: emscripten build failed [i686]

### Problem description

[emscripten.txt](https://github.com/termux/termux-packages/files/9685202/emscripten.txt)

/llvm/ADT/PointerUnion.h:19:
/home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/llvm/include/llvm/ADT/PointerIntPair.h:60:47: error: no member named 'getPointer' in 'llvm::PointerIntPairInfo<clang::Module *, 3, llvm::PointerLikeTypeTraits<clang::Module *>>'
  PointerTy getPointer() const { return Info::getPointer(Value); }
                                              ^
/home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/clang/include/clang/Lex/ModuleMap.h:172:48: note: in instantiation of member function 'llvm::PointerIntPair<clang::Module *, 3, clang::ModuleMap::ModuleHeaderRole>::getPointer' requested here
    Module *getModule() const { return Storage.getPointer(); }
                                               ^
In file included from /home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/clang/lib/Lex/PPDirectives.cpp:15:
In file included from /home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/clang/include/clang/Basic/FileManager.h:18:
In file included from /home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/clang/include/clang/Basic/FileEntry.h:21:
In file included from /home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/llvm/include/llvm/ADT/PointerUnion.h:19:
/home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/llvm/include/llvm/ADT/PointerIntPair.h:62:50: error: no member named 'getInt' in 'llvm::PointerIntPairInfo<clang::Module *, 3, llvm::PointerLikeTypeTraits<clang::Module *>>'
  IntType getInt() const { return (IntType)Info::getInt(Value); }
                                                 ^
/home/builder/.termux-build/emscripten/cache/llvm-project-8b587113b746f31b63fd6473083df78cef30a72e/clang/include/clang/Lex/ModuleMap.h:175:55: note: in instantiation of member function 'llvm::PointerIntPair<clang::Module *, 3, clang::ModuleMap::ModuleHeaderRole>::getInt' requested here
    ModuleHeaderRole getRole() const { return Storage.getInt(); }
                                                      ^
1 warning and 4 errors generated.

### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/packages/emscripten/build.sh
+++ b/packages/emscripten/build.sh
@@ -3,6 +3,7 @@ TERMUX_PKG_DESCRIPTION="Emscripten: An LLVM-to-WebAssembly Compiler"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@truboxl"
 TERMUX_PKG_VERSION="3.1.23"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/emscripten-core/emscripten.git
 TERMUX_PKG_GIT_BRANCH=$TERMUX_PKG_VERSION
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
@@ -52,8 +53,8 @@ opt/emscripten/LICENSE
 
 # https://github.com/emscripten-core/emscripten/issues/11362
 # can switch to stable LLVM to save space once above is fixed
-_LLVM_COMMIT=8b587113b746f31b63fd6473083df78cef30a72e
-_LLVM_TGZ_SHA256=15490ea18bd050b35dcbd036b60fae1c5f16a9389f95b1a4faca29d2cbf64d94
+_LLVM_COMMIT=945a1468c922573a07b334a130d05f0ecca40926
+_LLVM_TGZ_SHA256=26d6d4ba7844de81ae600d6f804fb047809079925072a557bc3faf4a5d4b0b48
 
 # https://github.com/emscripten-core/emscripten/issues/12252
 # upstream says better bundle the right binaryen revision for now

